pkgs/tools/misc/iay/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, nixosTests
+, openssl
+, pkg-config
+, Security
+, Foundation
+, Cocoa
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "iay";
+  version = "0.4.0";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    Security
+    Foundation
+    Cocoa
+  ]
+  ++ [ openssl pkg-config ]
+  ;
+
+  NIX_LDFLAGS = lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [ "-framework" "AppKit" ];
+
+  src = fetchFromGitHub {
+    owner = "aaqaishtyaq";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0r2yp34gxkh32amrysfj1jg543dh0kyqxzcx0zyi6a8y9232d8ky";
+  };
+
+  cargoHash = "sha256-SMqiwM6LrXXjV4Mb2BY9WbeKKPkxiYxPyZ4aepVIAqU=";
+
+  meta = with lib; {
+    description =
+      "iay! The minimalistic, blazing-fast, and extendable prompt for bash and zsh.";
```

```suggestion
    description = "Minimalistic, blazing-fast, and extendable prompt for bash and zsh";
```

short descriptions should never be self-referent and should never end with punctuation marks

---

Done. Thank you.